src/main/java/UseCase/createPlaylist.java

```diff
@@ -5,27 +5,56 @@
+import OutputBoundary.newPlaylistOutputBoundary;
+import OutputData.newPlaylistOutputData;
+
+//import OutputBoundary.newPlaylistOutputBoundary;
 
+public class createPlaylist implements newPlaylistInputBoundary {
+        private newPlaylistOutputBoundary outputBoundary;
+        private String message;
+    public createPlaylist(newPlaylistOutputBoundary outputBoundary){
+        this.outputBoundary = outputBoundary;
+        this.message = "";
+    }
+//    private final newPlaylistOutputBoundary newPlaylistOutputBoundary;
+//    public createPlaylist(newPlaylistOutputBoundary newPlaylistOutputBoundary){
+//        this.newPlaylistOutputBoundary = newPlaylistOutputBoundary;
+//    }
     /**
-     * @param playlistInputData holds necessary data to instantiate a new playlist
+     * @param newplaylistInputData holds necessary data to instantiate a new playlist
      * if hasFirstSong() is false, then newPlaylist() will instantiate an empty playlist and vice versa if it's value
      * is true
      */
-    public String newPlaylist(newPlaylistInputData playlistInputData) {
+    public void newPlaylist(newPlaylistInputData newplaylistInputData) {
         Playlist playlist;
         playlistAccessInterface library = playlistLibrary.getInstance();
-        if (!playlistInputData.hasFirstSong()) {
-            playlist = new Playlist(playlistInputData.getId(), playlistInputData.getPlaylistName(), playlistInputData.getOwner()
-                    );
-
+        if (!newplaylistInputData.hasFirstSong()) {
+            playlist = new Playlist(newplaylistInputData.getId(), newplaylistInputData.getPlaylistName(),
+                    newplaylistInputData.getOwner());
         }
         else{
-            playlist = new Playlist(playlistInputData.getId(), playlistInputData.getPlaylistName(), playlistInputData.getOwner(),
-                     playlistInputData.getFirstSong());
+            playlist = new Playlist(newplaylistInputData.getId(), newplaylistInputData.getPlaylistName(), newplaylistInputData.getOwner(),
+                     newplaylistInputData.getFirstSong());
         }
+        newplaylistInputData.getOwner().addPlaylist(playlist);
+        // save the newly created playlist to the RegularUser's playlist list
         library.savePlaylist(new playlistDsData(playlist));
-        return "Playlist created!";
+
+        //TODO #1: new potential implementation of sending confirmation message via outputdata, check correctness
+        //TODO #2: Regain will to live
+
+        //generate output data
+        newPlaylistOutputData outputData = new newPlaylistOutputData("Playlist created!");
+        // this.newPlaylistOutputBoundary.playlistCreationConfirmation(outputData);
+        // this.outputBoundary.getPlaylistCreationConfirmation(outputData);
+
+        //set message attribute to output data
+        this.message = this.outputBoundary.getPlaylistCreationConfirmation(outputData);
```

Comment:
why is the outputBoundary / presenter returning information to the use case? it should be fine to call the presenter's function and not store the resulting message in the use case interactor.